tests/tests/src/record_regression.mjs

```diff
@@ -171,36 +172,24 @@ function inlinedRecord(ir) {
   let x1 = ir.x1;
   let x0 = ir.x0;
   if (x1 !== undefined) {
-    switch (x1) {
-      case "x1" :
-        let x2 = ir.x2;
-        if (x2 !== undefined) {
-          return [
-            x0,
-            "x1",
-            x2,
-            ir.x3
-          ];
-        }
-        break;
-      case "xx1" :
```

Comment:
This case is now (correctly) optimized away, right?

Reply:
Not quite.
Fixed.